packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -41,7 +41,6 @@ const GridTab: FC<GridTabProps> = memo(({ isActive, model, onSelect, pullRight,
     const className = classNames({
         active: isActive,
         'pull-right': pullRight,
-        'no-data': showRowCount && !rowCount,
```

Comment:
This will affect all usages of the TabbedGridPanel, as I'm sure you are aware. Just doubling checking that we aren't trying to target just the Sample Finder case. I think this change makes sense across the board, but want to make sure.

Reply:
I think it makes sense across the board too. There's nothing really special about the Sample Finder grids in this regard.